<launch>
  <node name="miivii_detector" pkg="miivii_detector" type="miivii_detector_node" output="screen" >
    <param name="thresh" value="0.25"/>
    <param name="nms" value="0.45"/>

    <param name="camera_count" value="4" />

    <!-- param, gmsl camera topic -->
    <param name="camera1_topic" value="/camera_perception/perception/front/image_raw" />
    <param name="camera1_detect_results" value="/camera_perception/perception/front/detection_results" />
    <param name="camera1_with_rect" value="/camera_perception/perception/front/image_raw_draw_rect" />

    <param name="camera2_topic" value="/camera_perception/perception/left/image_raw" />
    <param name="camera2_detect_results" value="/camera_perception/perception/left/detection_results" />
    <param name="camera2_with_rect" value="/camera_perception/perception/left/image_raw_draw_rect" />

    <param name="camera3_topic" value="/camera_perception/perception/right/image_raw" />
    <param name="camera3_detect_results" value="/camera_perception/perception/right/detection_results" />
    <param name="camera3_with_rect" value="/camera_perception/perception/right/image_raw_draw_rect" />

    <param name="camera4_topic" value="/camera_perception/perception/rear/image_raw" />
    <param name="camera4_detect_results" value="/camera_perception/perception/rear/detection_results" />
    <param name="camera4_with_rect" value="/camera_perception/perception/rear/image_raw_draw_rect" />

    <param name="publish_result_image" value="false" />
    <param name="debug_time" value="true" />
  </node>
</launch>
